main.py

```diff
@@ -241,22 +245,24 @@ async def fill_blacklist(self) -> None:
         self.blacklist = {r["snowflake_id"] for r in records}
 
     async def get_prefix(self, message: discord.Message) -> Union[List[str], str]:
-        """Handles custom prefixes, this function is invoked every time process_command method is invoke thus returning
+        """Handles custom prefixes, this function is invoked every time process_command method is invoked thus returning
```

Comment:
honestly, this is just some grammar error at this point can u change it to
"Handles custom prefixes, this function gets invoke every time bot.get_context was invoked"

Reply:
fixed